assets/images/logo/logo.svg

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Did you accidentally hit Save in Inkscape? Or are there any real changes?

Reply:
No real changes.